src/options.c

```diff
@@ -320,6 +321,7 @@ options_init(options_t *options)
 	options->provider_args = NULL;
 
 	options->use_fade = -1;
+    options->fade_duration = -1;
```

Comment:
It seems like you used whitespaces here but the code above/below uses a tab.

Reply:
Oh, well spotted, thanks. I'll fix that.

Reply:
Fixed!